meta-xt-driver-domain/recipes-connectivity/xen-network/xen-network.bb

```diff
@@ -8,7 +8,7 @@ SRC_URI = " \
     file://bridge-up-notification.service \
     file://eth0.network \
     file://xenbr0.netdev \
-    file://60-xenbr0.network \
```

Comment:
Hi @lorc, for virtio I've added an additional networkd configuration file - vif-emu.network.

I've started with the following combination of file names:

- vif-emu.network
- 60-xenbr0.network

And it didn't work for NFS DomU. I didn't get 192.168.0.1/24 IP address assigned to xenbr0, when vif-emu interface appeared on the list after the start of the QEMU. That has led to a failing NFS boot of the virtio DomU domain.

I assumed that there was some ordering issue, and renamed files to explicitly mention the order of operations:

- 60-vif-emu.network
- 50-xenbr0.network

After that issue disappeared.

Change of names does not have an impact on the order of execution of these 2 files. In both cases '50-xenbr0.network' is considered with the higher priority. But, probably, when the 60-vif-emu.network is used with the digit prefix - it is executed with a higher priority, than some other existing network settings. And that does the trick.

Probably, when used without the prefix, vif-emu.network, which starts from 'v', is applied before any xenbr0-related networkd files, which start from 'x' and have less priority. And that leads to issues.

And if I needed to set priority to vif-emu.network, it should be still less prior than 60-xenbr0.network. I could use 70, but it is already occupied by 70-eth0.network ( after it is copied into the image ). Thus, I've decided to place vif-emu.network on prio 60 and move xenbr0.network to prior 50 to avoid any collisions and still preserve the "most probably working" order.

Reply:
You need to document this in the commit message, not in the comment. Probably easier solution were to name your config "65-vif-emu.network". But I am okay with "xenbr0" renaming, just describe  it properly.

Reply:
Hi @lorc, accepted. I'll change the commit message.

Reply:
Hi @lorc, done.